bot_modules/http_server.js

```diff
@@ -55,6 +56,8 @@ module.exports = {
 						return;
 					}
 					// check for and run command
+					console.log('run command');
+					console.log(commands);
```

Comment:
not sure why we would want to console log the list of commands every time a command is called
something more like `'run command: ' + data.command` would make more sense

Reply:
I was just transferring over what I saw in the commits, but I can do that